### PR TITLE
Added a time remaining count down for aby zones

### PR DESCRIPTION
**Pull request by smd111**

### CODE REVIEW

**Comment by smd111** [comment]
its a little off because the countdown gets rest ever time the time remaining gets updated but it has to do this or the timer will get vary far off

File: addons/pointwatch/FIXING POINTWATCH.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@ Method 1 - POLUtils
 5) Wait for it to load, then export the entire table as xml
 6) Open the xml file up in a text editor, like Notepad or Notepad++.
 7) Control-F for the key phrase.
+ --look for: ≺Set Color #5≻Visitant Light Intensity≺Set Color #1≻
```
Comment (z16) [inline addons/pointwatch/FIXING POINTWATCH.txt]: Would remove that, it's a very specific example, and the `<Set Color #x>` stuff will probably only confuse people.

Comment (smd111) [inline addons/pointwatch/FIXING POINTWATCH.txt]: if you search for "≺Set Color #5≻Visitant Light Intensity≺Set Color #1≻" its exactly what you want for the offset

**Comment by smd111** [comment]
any news on if this will be accepted??

**Comment by smd111** [comment]
thx byrth

**Pull request merged**